src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/LanguageServices/VsContainedLanguageComponentsFactoryBase.cs

```diff
@@ -49,6 +67,8 @@ internal abstract class VsContainedLanguageComponentsFactoryBase : IVsContainedL
             {
                 await ProjectServices.ThreadingService.JoinableTaskFactory.SwitchToMainThreadAsync();
 
+                await InitializeAsync().ConfigureAwait(true);
```

Comment:
Does this need to be called from the main thread, since we're awaiting anyway? It might be better to switch to the main thread after initializing.